### ISSUE TITLE
Can't click on Ui-info-suite options or icons

### ISSUE THREAD

**Issue by purpleturtle4296** [cdaragorn/Ui-Info-Suite#78]
When clicking on any option of the ui-info menu it simply doesn't work. I tried disabling every single mod except for smapi and UIif and even so I got no response :(
I can't click on the menu nor on the billboard or calendar, and can't uncheck the "show on map" for NPCs (which was the thing I wanted to do the most so I could use a similar mod).
Smapi and UI are updated.

https://user-images.githubusercontent.com/87454996/125713156-ccd6090e-320b-48ad-8689-502e9d7588a6.mp4

**Comment by ArtySh0ck** [comment]
Yeah, i think thats because some rendering problem. When i try to set up Resizing World in settings back to default 100% - its works great, but if u need to play 75% as me - that's problem =(((

**Comment by exedox** [comment]
same here :(
